### PR TITLE
Brush up some sentences

### PR DESCRIPTION
**Pull request by hkurokawa**
This PR improves some description of README from English point of view.

Find my comments for the changes to see why I believe they can be improved.

Thank you.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,18 +3,16 @@
 Retainer is an Android library that helps you retain objects in an Activity / Fragment during configuration changes.
 
 It auto-generates a Fragment internally by using annotation processing.
-A generated Fragment is called `setRetainInstance(true);` in its `onCreate()` to preserve stateful objects.
```
Comment (hkurokawa) [inline README.md:6]: I'm afraid "A fragment is called a method" implies "the name of the fragment is the method".

File: README.md [hunk 2]
```diff
@@ -3,18 +3,16 @@
...
-If you need more information about the handling of configuration changes, please check the following document.
+If you need more information about the handling of configuration changes, check the following document.
 
 https://developer.android.com/guide/topics/resources/runtime-changes.html
 
 ## How to use
 
 ### Activity
 
-Supports only `android.support.v4.app.FragmentActivity` or its subclasses.
```
Comment (hkurokawa) [inline README.md]: Probably, this makes sense only for Fragments.

File: README.md [hunk 3]
```diff
@@ -3,18 +3,16 @@
 Retainer is an Android library that helps you retain objects in an Activity / Fragment during configuration changes.
 
 It auto-generates a Fragment internally by using annotation processing.
-A generated Fragment is called `setRetainInstance(true);` in its `onCreate()` to preserve stateful objects.
+`setRetainInstance(true)` is called for the generated Fragment in its `onCreate()` so that the Fragment preserves stateful objects for the Activity.
 
-If you need more information about the handling of configuration changes, please check the following document.
+If you need more information about the handling of configuration changes, check the following document.
```
Comment (hkurokawa) [inline README.md:8]: This is just my preference. "please" sounds redundant. Ignore this change if you do not like this.

File: README.md (hunk 1 above)
Comment (nakamuuu) [reply README.md:6]: Oh, that's a good point😅

**Comment by nakamuuu** [comment]
LGTM! 👍

**Pull request merged**